keras_preprocessing/image/image_data_generator.py

```diff
@@ -652,6 +649,9 @@ def flow_from_dataframe(self,
             warnings.warn('sort is deprecated, batches will be created in the'
                           'same order than the filenames provided if shuffle'
                           'is set to False.')
+        if 'drop_duplicates' in kwargs:
+            warnings.warn('drop_duplicates is deprecated.', DeprecationWarning)
```

Comment:
The user would probably like some indications: How can they remove duplicate themselves? Using df.drop_duplicates.

LGTM otherwise

Reply:
done